pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -884,6 +885,13 @@ class Timestamp(_Timestamp):
             - 'NaT' will return NaT for an ambiguous time
             - 'raise' will raise an AmbiguousTimeError for an ambiguous time
 
+        nonexistent : 'shift', 'NaT', default 'raise'
```

Comment:
hmm, this is now a bit confusing, e.g. there is overlap between errors and nonexistent. Is it possible to use errors instead here? (and not add a new keyword)?